client/extensions/wp-super-cache/caching.jsx

```diff
@@ -60,23 +59,23 @@ const Caching = ( {
 					<FormFieldset className="wp-super-cache__cache-type-fieldset">
 						<FormLabel>
 							<FormRadio
-								checked={ !! is_super_cache_enabled && !! cache_mod_rewrite }
+								checked={ 'mod_rewrite' === cache_type }
```

Comment:
Nitpick -- [no need for yoda style in Calypso](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#yoda-conditions) :slightly_smiling_face: